mslib/msui/__init__.py

```diff
@@ -180,7 +180,7 @@ class MissionSupportSystemDefaultConfig(object):
     mscolab_server_url = "http://localhost:8083"
     # ToDo refactor to rename this to data_dir/mss_data_dir
     # mss dir
-    mss_dir = "~/mss"
+    mscolab_data_dir = os.path.join(os.path.expanduser("~/"), ".local", "share", "mss")
```

Comment:
see above, the dirs have to be verified to exists on all platforms

Reply:
I do not know how to verify the existance of directories with fs without using os.path.